widgets/common/src/filters/RuleOperatorDropdown.tsx

```diff
@@ -0,0 +1,20 @@
+import type { FunctionComponent } from 'react';
+
+const operators = [
+    { name: 'any_of', label: 'is one of' },
+    { name: 'not_any_of', label: 'is not one of' },
+    { name: 'is_null', label: 'key is not' },
+    { name: 'is_not_null', label: 'key is' },
+    { name: 'contain', label: 'contains' },
+    { name: 'not_contain', label: 'does not contain' },
+    { name: 'start_with', label: 'starts with' },
+    { name: 'end_with', label: 'ends with' }
+];
+
+const RuleOperatorDropdown: FunctionComponent = () => {
+    const { Dropdown } = Stage.Basic;
+    const operatorsOptions = _.map(operators, operator => ({ text: operator.label, value: operator.name }));
```

Comment:
How about avoiding lodash :slightly_smiling_face: ?

```suggestion
    const operatorsOptions = operators.map(operator => ({ text: operator.label, value: operator.name }));
```

Also, let's make sure that the objects returned match what the `Dropdown` expects as `options`:

```suggestion
    const operatorsOptions = operators.map((operator): DropdownItemProps => ({ text: operator.label, value: operator.name }));
```

btw. you can perhaps see now why I skipping parentheses around arrow function parameters is not helpful when using TS - you need to add them each time you want to either add a parameter, add type annotation for a parameter, or add a type annotation for the return type :slightly_smiling_face: Generally arrow function parameter are always enforced in TS projects, but I don't see the need to add them in this one, as that would require lots of changes :slightly_smiling_face:

Reply:
Thanks for the suggestions. Removed lodash and added type annotation.